### PR TITLE
[QA] fix zapwallettxes test

### PR DESCRIPTION
**Pull request by jonasschnelli**
- remove shell test script and add a python equivalent

Wallet tests (we need more!) must be available in case of a upcoming wallet file format change.

fix for #5609

### CODE REVIEW

File: qa/pull-tester/rpc-tests.sh [hunk 1]
```diff
@@ -26,17 +26,24 @@ testScripts=(
...
         then
-            echo -e "running testscript \033[1m${testScripts[$i]}...\033[0m"
+            found=1
+            echo -e "Running testscript \033[1m${testScripts[$i]}...\033[0m"
             ${BUILDDIR}/qa/rpc-tests/${testScripts[$i]} --srcdir "${BUILDDIR}/src"
         fi
     done
+    if [ $found == 0 ]
+    then
+        echo -e "\033[1mTest not found\033[0m"
```
Comment (jonasschnelli) [inline qa/pull-tester/rpc-tests.sh]: smuggled in: give simple output when selected test script was not available

Comment (jonasschnelli) [inline qa/pull-tester/rpc-tests.sh]: Had to remove this because it had break travis.

**Comment by fanquake** [comment]
Related #4071. Almost every test is python now, bar conflictedbalance.sh

**Comment by laanwj** [comment]
Tested ACK

**Pull request merged**